cypress/support/commands.js

```diff
@@ -521,69 +350,31 @@ Cypress.Commands.add(
 	}
 );
 
-Cypress.Commands.add('inWhichYearAreYouStudying', (stage) => {
+Cypress.Commands.add("inWhichYearAreYouStudying", (stage) => {
 	switch (stage) {
```

Comment:
This switch statement is redundant as we pass the stage, so it can become:

```
Cypress.Commands.add("inWhichYearAreYouStudying", (stage) => {
  cy.contains(stage).click();
});
```

Reply:
Resolved